Client/Frontend/Library/HistoryPanel/HistoryPanelV2.swift

```diff
@@ -595,8 +595,9 @@ extension HistoryPanelV2 {
             header.textLabel?.textColor = UIColor.theme.tableView.headerTextDark
             header.contentView.backgroundColor = UIColor.theme.tableView.selectedBackground
             header.textLabel?.text = actualSection.title // At worst, we have a header with no text.
-            
-            let historySectionsWithGroups = viewModel.searchTermGroups.map { group in
+
+            // let historySectionsWithGroups
+            let _ = viewModel.searchTermGroups.map { group in
                 viewModel.groupBelongsToSection(asGroup: group)
             }
```

Comment:
I didn't know if it could entirely be commented. Easy solution for now was just to comment out the warning cause itself so it doesn't appear each time I'm building 😝 When it will be used that part can be put back (hence why I left the name commented instead of removing it)